### ISSUE TITLE
TextField won't respond in some cases simply by adding swiperefresh 0.21.3-beta

### ISSUE THREAD

**Issue by harrynull** [google/accompanist#888]
## Describe the bug

TextField and Buttons (and presumably many other things that respond to click events) fail (not responding) after you click any empty place in a Column.

To avoid confusions, I know the following code does not contain anything relevant to accompanist, but the bug will only occur when swiperefresh:0.21.3-beta is added into your dependency (and **not using it at all**!):
```
    implementation "com.google.accompanist:accompanist-swiperefresh:0.21.3-beta"
```

As bizarre as that might sound, since it is only happening when I am using this library, I am submitting the bug report here.

## To Reproduce
1. Code
```kotlin
class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            var text by remember { mutableStateOf("initial")}
            Column(modifier = Modifier.fillMaxSize()) {
                BasicTextField(
                    value = text,
                    onValueChange = { text = it }
                )
            }
        }
    }
}
```

2. Click on any empty place first (i.e. anywhere but the textfield)

3. Click on the text field

4. Nothing happens

## Expected behavior
TextField and buttons should respond and work properly

## Environment:
 - Accompanist version: **0.21.3-beta**
- Jetpack Compose release version: 1.1.0-beta03
- Android Studio Build: Arctic Fox 2020.3.1 Patch 3 Build #AI-203.7717.56.2031.7784292
- Kotlin version: 1.5.31
- Android OS version: 11

## Additional context
1. It works until you click on any empty place
2. It works without that `Modifier.fillMaxSize()`
3. 0.20.2 works fine

**Comment by travissmithihr** [comment]
I think this is an issue with Jetpack Compose 1.1.0-beta03.  Here's a link to their issue tracker where someone else reported the same problem: https://issuetracker.google.com/issues/207222640

**Comment by harrynull** [comment]
@travissmithihr I think you are right! Thank you for pointing me to that ticket. I think it is indeed the problem with Compose 1.1.0-beta3.
I tried again and it seems that 1.1.0-beta3 alone without accompanist could trigger this issue (although not sure why it worked earlier). And when I tested with 1.0.5 + 0.21.3-beta, it is apparently because of transitive dependency making it use 1.1.0-beta3 rather than 1.0.5 even though I explicitly specify the latter.
Closing due to not an issue with accompanist.
